helpers/cdisHelper.js

```diff
@@ -17,6 +17,15 @@ class CDISHelper extends Helper {
       testResult.err.stack += res;
     });
   }
+
+  noTimeoutEnter() {
```

Comment:
this function should probably be in the API utils file

Reply:
I feel like this function should be in a Helper file since it is customizing the `WebDriverIO helper`. Although it might be good if I move it to a separate `exportToWorkspaceHelper` file?

Reply:
right now the API utils file contains anything that doesn't fit in another utils 😆but it's fine if you want to keep this function here